chatmaild/src/chatmaild/doveauth.py

```diff
@@ -45,23 +46,30 @@ def is_allowed_to_create(config: Config, user, cleartext_password) -> bool:
         return False
     localpart, domain = parts
 
+    if localpart == "echo":
+        # echobot account should not be created in the database
+        return False
+
     if (
         len(localpart) > config.username_max_length
         or len(localpart) < config.username_min_length
     ):
-        if localpart != "echo":
-            logging.warning(
-                "localpart %s has to be between %s and %s chars long",
-                localpart,
-                config.username_min_length,
-                config.username_max_length,
-            )
-            return False
+        logging.warning(
+            "localpart %s has to be between %s and %s chars long",
+            localpart,
+            config.username_min_length,
+            config.username_max_length,
+        )
 
     return True
 
 
 def get_user_data(db, config: Config, user):
+    if user == f"echo@{config.mail_domain}":
+        return dict(
+            home=f"/home/vmail/mail/{config.mail_domain}/echo", uid="vmail", gid="vmail"
```

Comment:
```suggestion
            home=f"/home/vmail/mail/{config.mail_domain}/echo@{config.mail_domain}", uid="vmail", gid="vmail"
```

Typically, the vmail directory would be `/echo@nine.testrun.org,` not `/echo` I think...